tests/test_config.py

```diff
@@ -17,3 +18,15 @@
 def test_config(filename):
     c = Config.load_yaml(str(DATA / '{}.yaml'.format(filename)))
     assert isinstance(c, dict)
+
+
+@pytest.mark.parametrize("filename", [
+    ('config_import_without_comment'),
+    ('config_import_with_comment'),
+])
+def test_config_comment_before_import_handle(filename):
+    start_wd = os.getcwd()
+    os.chdir('{}/data'.format(Path(__file__).parent))
```

Comment:
You should use [monkeypatch.chdir](https://docs.pytest.org/en/latest/monkeypatch.html) instead of manual changes. 

In current implementation if test failed in the middle, current directory will not be changed to previous.